<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessManage</class>
 <widget class="QMainWindow" name="ProcessManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessManage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="whatsThis">
      <string/>
     </property>
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>541</width>
        <height>241</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>539</width>
         <height>239</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-10</y>
          <width>541</width>
          <height>271</height>
         </rect>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>进程号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CPU</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>占用内存</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起始时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>任务描述</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>结束进程</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>系统性能</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>物理内存：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>已使用内存：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU使用率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>开机时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>进程数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>线程数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>用户</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>541</width>
        <height>211</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>539</width>
         <height>209</height>
        </rect>
       </property>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>541</width>
          <height>211</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>用户</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>状态</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>标号</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>93</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>断开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>93</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>注销</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>任务管理（&amp;F）</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_N"/>
    <addaction name="separator"/>
    <addaction name="action_C"/>
    <addaction name="separator"/>
    <addaction name="action_M"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_U">
    <property name="title">
     <string>用户管理（&amp;U）</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="action_Q_2"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助（&amp;H）</string>
    </property>
    <addaction name="action_S"/>
    <addaction name="action_I"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_U"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="text">
    <string>新建任务（&amp;N）</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>结束任务（&amp;C）</string>
   </property>
  </action>
  <action name="action_M">
   <property name="text">
    <string>任务描述（&amp;M）</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>任务切换（&amp;Q）</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>切换用户（&amp;H）</string>
   </property>
  </action>
  <action name="action_Q_2">
   <property name="text">
    <string>退出（&amp;E）</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>使用说明（&amp;S）</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>开发信息（&amp;I）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
